.github/workflows/db2graph_test_postgres.yml

```diff
@@ -0,0 +1,49 @@
+name: Testing DB2GRAPH using postgres
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  # Label of the container job
+  container-job:
+    # Containers must run in Linux based operating systems
+    runs-on: ubuntu-latest
+    # Docker Hub image that `container-job` executes in
+    container: python:3.6.15
```

Comment:
Would be good to use additional versions of python for these tests. I believe it's pretty easy to do with these workflows. Add 3.7, 3.8, 3.9 and 3.10

Reply:
Updated the workflow